src/ORM/FieldType/DBTime.php

```diff
@@ -26,33 +27,30 @@ class DBTime extends DBField
 {
     /**
      * Standard ISO format string for time in CLDR standard format
+     * This is equivalent to php date format "H:i:s" e.g. 09:30:00
      */
     public const ISO_TIME = 'HH:mm:ss';
 
+    private static array $field_validators = [
+        TimeFieldValidator::class,
+    ];
+
     public function setValue(mixed $value, null|array|ModelData $record = null, bool $markChanged = true): static
     {
         $value = $this->parseTime($value);
```

Comment:
```suggestion
       parent::setValue($this->parseTime($value), $record, $markChanged);
```
In many of the other implementations you've updated to explicitly call `parent::setValue()` - seems we should do so here, too.

Reply:
I've largely reverted this class to how it was in CMS 5